lib/mix/tasks/nerves/new.ex

```diff
@@ -4,7 +4,8 @@ defmodule Mix.Tasks.Nerves.New do
 
   @nerves Path.expand("../../../..", __DIR__)
 
-  @nerves_vsn "1.3"
+  # @nerves_vsn "1.3"
+  @nerves_dep ~s[{:nerves, github: "nerves-project/nerves", branch: "elixir-18", override: true, runtime: false}]
```

Comment:
This will need to be changed once the other PR is released right?

Reply:
right, 🐔 🥚